clusterloader2/pkg/measurement/common/simple/wait_for_pods.go

```diff
@@ -139,7 +139,7 @@ func waitForPods(clientSet clientset.Interface, namespace, labelSelector, fieldS
 			if log {
 				glog.Infof("%s: %s: %s", callerName, selectorsString, podsStatus.String())
 			}
-			if podsStatus.Running == desiredPodCount {
+			if podsStatus.Running == desiredPodCount && len(pods) == desiredPodCount {
```

Comment:
nit: can we change to:

len(pods) == podsStatus.Running && podsStatus.Running == desiredPodCount

This is more intuitive (all pods are running and there is correct number of running pods).

[I know it's technically the same, but it's easier to reason about.]

Reply:
Done.